benchmarks/src/bin/h2o.rs

```diff
@@ -113,13 +118,16 @@ async fn group_by(opt: &GroupBy) -> Result<()> {
     let start = Instant::now();
     let df = ctx.sql(sql).await?;
     let batches = df.collect().await?;
-    let elapsed = start.elapsed().as_millis();
-
+    let elapsed = start.elapsed().as_secs_f64() * 1000.0;
+    let numrows = batches.iter().map(|b| b.num_rows()).sum::<usize>();
```

Comment:
Thanks @jaylmiller for looking into this. 

Noticed for other testcases you calc numrows before elapsed, perhaps to prevent numrows runtime to be part of benchmark  runtime

Reply:
Yes! Good catch thank youi... was a mistake by me

Reply:
Another thing I'm thinking is can it be calculating num rows triggers some system cache and benchmark will run faster, alhough its unexpected

Reply:
I think num_rows is pretty fast (it doesn't actually do any work , it just returns a field's value): https://docs.rs/arrow-array/35.0.0/src/arrow_array/record_batch.rs.html#278